src/Extensions/Hosting.Services.Web/Middlewares/UserIdentity/IdentityProviders/IpBasedUserIdentityProvider.cs

```diff
@@ -10,16 +10,32 @@ namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
 {
 	internal class IpBasedUserIdentityProvider : IUserIdentityProvider
```

Comment:
Is the intent that we provide 2 providers for full or limited IP address (last two places zeroed out) or should this provider have that as configuration?
On a side note, did we ever use the full IP for generating the hash in the old code base or did we always use the IP with the last two positions zeroed?